### PR TITLE
DEV-143: Drawing In Air

### PR DESCRIPTION
**Pull request by PrestonB1123**
Removed drawing in air while in desktop mode.
https://highfidelity.atlassian.net/browse/DEV-143
also resolves: https://highfidelity.atlassian.net/browse/BUGZ-535

### CODE REVIEW

File: usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereClient.js [hunk 1]
```diff
@@ -335,11 +334,6 @@
             wasLastPointOnBoard = isCurrentPointOnBoard;
             if (event.isLeftButton) {
                 drawingInDesktop = true;
-                if (!isCurrentPointOnBoard) {
```
Comment (zfox23) [inline usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereClient.js]: I don't _think_ this is the right solution - I think you simply want to return early if `!isCurrentPointOnBoard`. No?

File: usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereClient.js [hunk 2]
```diff
@@ -364,11 +358,6 @@
             currentStrokeWidth = _this.getCurrentStrokeWidth();
             var isCurrentPointOnBoard = _this.maybeProjectPointOntoBoard(whiteBoardIntersectionData, true);
             if (event.isLeftButton) {
-                if (!isCurrentPointOnBoard) {
```
Comment (zfox23) [inline usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereClient.js]: I don't _think_ this is the right solution - I think you simply want to return early if `!isCurrentPointOnBoard`. No?

File: usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereClient.js (hunk 1 above)
Comment (PrestonB1123) [reply usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereClient.js]: Ack

File: usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereClient.js (hunk 2 above)
Comment (PrestonB1123) [reply usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereClient.js]: Ack

**Pull request merged**